week6/homework/Product.cs

```diff
@@ -0,0 +1,55 @@
+using Dapper;
+using MySql.Data.MySqlClient;
+
+namespace HackYourFuture.Week6
+{
+    public interface IProductRepository
+    {
+        Task<IEnumerable<Product>> GetProducts();
+        Task<Product> PostProduct(Product productPost);
+        Task<Product> PutProduct(Product product);
+        Task<int> DeleteProduct(int id);
+    }
+
+    public class ProductRepository : IProductRepository
+    {
+        private string connectionString;
+
+        public ProductRepository(IConfiguration configuration)
+        {
+            this.connectionString = configuration.GetConnectionString("Default");
+        }
+
+        public async Task<IEnumerable<Product>> GetProducts()
+        {
+            using var connection = new MySqlConnection(connectionString);
+            var products = await connection.QueryAsync<Product>("SELECT id,name,price FROM products");
+            return products;
+        }
+
+        public async Task<Product> PostProduct(Product product)
```

Comment:
A small suggestion here, a more appropriate name for this method would be `Create`. Repositories should have "CRUD" names - Create/Read/Update/Delete.